java/Cherepnin_Oleh/src/com/zoolatech/lecture10/tasks/FileReader.java

```diff
@@ -0,0 +1,43 @@
+package com.zoolatech.lecture10.tasks;
+
+import java.io.*;
+import java.util.Date;
```

Comment:
Please avoid using util.Date. Here is why https://programminghints.com/2017/05/still-using-java-util-date-dont/
LocalDateTime will be better